pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -115,13 +115,18 @@ func NewQueue(kubeClient client.Client, recorder events.Recorder, cluster *state
 	provisioner *provisioning.Provisioner,
 ) *Queue {
 	queue := &Queue{
-		RateLimitingInterface: workqueue.NewRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(queueBaseDelay, queueMaxDelay)),
-		providerIDToCommand:   map[string]*Command{},
-		kubeClient:            kubeClient,
-		recorder:              recorder,
-		cluster:               cluster,
-		clock:                 clock,
-		provisioner:           provisioner,
+		RateLimitingInterface: workqueue.NewRateLimitingQueueWithConfig(
+			workqueue.NewItemExponentialFailureRateLimiter(queueBaseDelay, queueMaxDelay),
+			workqueue.RateLimitingQueueConfig{
+				Name:            "disruptionQueue",
```

Comment:
```suggestion
				Name:            "disruption.queue",
```

I'd prefer to have a similar style to what the actual controller name is; though, since the controller itself is going to have a workqueue, we need to make sure that the name that we choose here doesn't conflict with the controller workqueue. Maybe that means renaming the controller name to something like `disruption.queue.reconciler` or naming the workqueue to something like `disruption.workqueue`?